README.md

```diff
@@ -1,6 +1,9 @@
 # OpenTelemetry .NET
 [![Gitter chat](https://img.shields.io/gitter/room/opentelemetry/opentelemetry-dotnet)](https://gitter.im/open-telemetry/opentelemetry-dotnet)
-[![Build Status](https://dev.azure.com/opentelemetry/pipelines/_apis/build/status/open-telemetry.opentelemetry-dotnet-myget-update?branchName=master)](https://dev.azure.com/opentelemetry/pipelines/_build/latest?definitionId=2&branchName=master)
+

```

Comment:
Remove this line, otherwise it'll give a line break (the badges will show on two lines instead of one).

Reply:
Line 3 needs to be removed as well.

Reply:
done